src/Service/S3/tests/Integration/S3ClientTest.php

```diff
@@ -352,6 +352,7 @@ public function testGetObjectConsistent(): void
         self::assertEquals('content', $result->getBody()->getContentAsString());
         self::assertEquals('content', \stream_get_contents($result->getBody()->getContentAsResource()));
         self::assertEquals('content', \implode('', \iterator_to_array($result->getBody()->getChunks())));
+        self::assertEquals('content', \implode('', \iterator_to_array($result->getBody()->getChunks())));
```

Comment:
Why do we need this a second time?

Reply:
the `ResponseBodyStream` cache previous iteration in a fallback variable. This second check assert that both original, and fallback return the right response.

I notice (because of another bug) that test failed only on the second call.

Reply:
added comment about that